hscontrol/templates/apple.go

```diff
@@ -0,0 +1,149 @@
+package templates
+
+import (
+	"fmt"
+
+	"github.com/chasefleming/elem-go"
+	"github.com/chasefleming/elem-go/attrs"
+)
+
+func Apple(url string) *elem.Element {
+	return HtmlStructure(
+		elem.Title(nil,
+			elem.Text("headscale - Apple")),
+		elem.Body(attrs.Props{
+			attrs.Style: bodyStyle.ToInline(),
+		},
+			headerOne("headscale: iOS configuration"),
+			headerTwo("GUI"),
+			elem.Ol(nil,
+				elem.Li(nil,
+					elem.Text("Install the official Tailscale iOS client from the"),
```

Comment:
The space between `from the` and the appstore link is missing:

![image](https://github.com/user-attachments/assets/512f0bb3-b94d-42b3-bdac-2a1210dc6ef0)